pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -1388,7 +1388,7 @@ func TestControllerCreateError(t *testing.T) {
 
 	newDeployment(t, fakedynamicclient.Get(ctx), testRevision+"-deployment", 3)
 
-	got := perrors.Cause(ctl.Reconciler.Reconcile(context.Background(), key))
+	got := errors.Unwrap(errors.Unwrap(ctl.Reconciler.Reconcile(context.Background(), key)))
```

Comment:
Can't we use the `Is` function here as well to handle the recursion?

Reply:
Let me check.